semantickernel-api/src/main/java/com/microsoft/semantickernel/services/OrderedAIServiceSelector.java

```diff
@@ -66,13 +66,15 @@ private static Map<String, PromptExecutionSettings> settingsFromFunctionSettings
         return null;
     }
 
-//    @Nullable
-//    @Override
-//    public <T extends AIService> AIServiceSelection<T> trySelectAIService(
-//            Class<T> serviceType,
-//            @Nullable KernelArguments arguments) {
-//        return selectAIService(serviceType, arguments.getPromptExecutionSettings());
-//    }
+    @Nullable
+    @Override
+    public <T extends AIService> AIServiceSelection<T> trySelectAIService(
+            Class<T> serviceType,
+            @Nullable KernelArguments arguments,
+            Map<Class<? extends AIService>, AIService> services) {
```

Comment:
What's the plan for using the services map?

Reply:
It's used in some the subclasses. Just mirroring the existing method without the function.